soc/wch/ch32v/qingke_v4/Kconfig.soc

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
 # SPDX-License-Identifier: Apache-2.0
 
-config SOC_SERIES_QINGKE_V4C
+config SOC_SERIES_QINGKE_V4
```

Comment:
Why was the 'C' dropped? I'm fine with this, but it's surprising.

Reply:
There was a lengthy discussion about how to reorganize things some time ago: https://github.com/zephyrproject-rtos/zephyr/pull/87345#discussion_r2011812193 and https://github.com/zephyrproject-rtos/zephyr/pull/87125#discussion_r2007567308

Reply:
All good then.